src/Symfony/Component/Serializer/Normalizer/NormalizerInterface.php

```diff
@@ -41,8 +43,8 @@ public function normalize(mixed $data, ?string $format = null, array $context =
     /**
      * Checks whether the given class is supported for normalization by this normalizer.
      *
-     * @param mixed       $data   Data to normalize
-     * @param string|null $format The format being (de-)serialized from or into
+     * @param TData|mixed   $data   Data to normalize
```

Comment:
why keep mixed?

Reply:
For `supportsNormalization()`, it might presumably receive all sorts of data (since it's testing for serialization support).

Reply:
I guess you could argue that `mixed` already contains `TData` as well, so we might as well just keep `mixed` as a type.